src/components/StudyInfo/CavaticaCounts.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {List, Icon, Popup, Label, Button} from 'semantic-ui-react';
+import {projectOptions} from '../../common/enums';
+
+const CavaticaCounts = ({projects, title}) => {
+  if (projects && projects.length > 0) {
+    const types = projects.map(({node: {projectType}}) => projectType);
+    const typeCounts = types.reduce((count, type) => {
+      count[type] = (count[type] || 0) + 1;
+      return count;
+    }, {});
+    return (
+      <List horizontal>
+        <List.Item
+          as={Link}
+          to={`/study/${title}/cavatica`}
+          onClick={e => e.stopPropagation()}
```

Comment:
since we removed the top-level link on the study card do we need to `stopPropagation` here?  asking because I'm not sure

Reply:
`onClick={e => e.stopPropagation()}` this part is added becuase in study table view, each row is clickable.